### PR TITLE
Creating Whylogs transformer + renderers

### PR DESCRIPTION
**Pull request by murilommen**
# TL;DR
It creates the whylogs plugin to extend Flyte's usage.  

## Type
 - [ ] Bug Fix
 - [ ] Feature
 - [x] Plugin

## Are all requirements met?

 - [x] Code completed
 - [ ] Smoke tested
 - [x] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Complete description
It creates a schema for "transforming" whylogs' `DatasetProfileView` objects to and from the FlyteSchema. This PR also creates two renderers with FlyteDeck, one for the Constraints report and the other for the SummaryDriftReport.

## Tracking Issue
N/A

## Follow-up issue
_NA_

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -30,4 +30,5 @@ docs/source/plugins/generated/
 .pytest_flyte
 htmlcov
 *.ipynb
-*dat
+.env
+*dat
```
Comment (eapolinario) [inline .gitignore:34]: nit: add a newline.
